imports/plugins/included/connectors-shopify/server/hooks/orders.js

```diff
@@ -0,0 +1,278 @@
+  shopifyOrder.shipping_address = shippingAddress;
+  shopifyOrder.customer = convertCustomer(billingAddress, order);
+  shopifyOrder.email = order.email;
+  const paymentType = order.billing[index].method;
+  if (paymentType === "credit" && order.billing[index].mode === "authorize") {
+    shopifyOrder.financial_status = "authorized";
+  } else {
+    shopifyOrder.financial_status = "paid";
+  }
+  shopifyOrder.id = order._id;
+  const itemsForShop = order.getItemsByShop()[shopId];
+  shopifyOrder.line_items = convertLineItems(itemsForShop, order);
+  shopifyOrder.phone = order.billing[index].address.phone;
+  shopifyOrder.source_name = "reaction_export";
+  shopifyOrder.subtotal_price = order.getSubtotalByShop()[shopId];
+  shopifyOrder.token = order._id;
+  shopifyOrder.total_discounts = order.getDiscountsByShop()[shopId];
+  shopifyOrder.total_line_item_price = order.getItemsByShop()[shopId].reduce((total, item) => {
+    return total + (item.variants.price * item.quantity);
+  }, 0);
+  shopifyOrder.total_price = order.getTotalByShop()[shopId];
+  shopifyOrder.total_tax = order.getTaxesByShop()[shopId];
+  shopifyOrder.total_weight = shopifyOrder.line_items.reduce((sum, item) => {
+    return sum + (item.grams * item.quantity);
+  }, 0);
+  return shopifyOrder;
+}
+
+/**
+ * @private
+ * @summary Normalize all weights to grams
+ * @param {Number} weight - the original weight value
+ * @param {String} shopId - the shop we are converting for
+ * @returns {Number} the normalized value
+ */
+function normalizeWeight(weight, shopId) {
+  // if weight is not grams, convert to grams if is grams just return
+  const shop = Shops.findOne(shopId);
+  const { baseUOM } = shop;
+  // Could've used switch, this was easier to read
+  // should these be a utilities module somewhere?
+  if (baseUOM === "g") {
+    return weight;
+  }
+  if (baseUOM === "lb") {
+    const grams = weight * 453.592;
+    return grams;
+  }
+
+  if (baseUOM === "oz") {
+    const grams = weight * 28.35;
+    return grams;
+  }
+
+  if (baseUOM === "kg") {
+    const grams = weight / 1000;
+    return grams;
+  }
+}
+
+/**
+ * @private
+ * @summary Convert individual line items of an order
+ * @param {Array} items - The items on the order object
+ * @param {Object} order - The order we are converting for
+ * @returns {Array} An array of converted line items
+ */
+function convertLineItems(items, order) {
+  const lineItems = items.map((item) => {
+    const lineItem = {};
+    lineItem.fulfillable_quantity = item.quantity;
+    lineItem.fulfillment_service = "manual";
+    lineItem.fullfillment_status = null;
+    if (item.parcel && item.parcel.weight) {
+      lineItem.grams = normalizeWeight(item.parcel.weight, item.shopId);
+    }
+    lineItem.id = item._id;
+    lineItem.product_id = item.productId;
+    lineItem.quantity = item.quantity;
+    lineItem.requires_shipping = item.product.requiresShipping;
+    // lineItem.sku = ??? Not sure what should be the SKU here
```

Comment:
Reaction has a SKU, but I'm not sure it's used or set currently.
https://github.com/reactioncommerce/reaction/blob/master/lib/collections/schemas/products.js#L300